### PR TITLE
FEND-670: Updating peer dependencies of eslint and eslint-plugin-react

### PR DESCRIPTION
**Pull request by Nayeemuddin-khan**
Issue solved for deprecated 'EcmaFeature'

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"name": "eslint-config-smartthings",
-	"version": "3.0.0",
+	"version": "3.3.0",
```
Comment (xres) [inline package.json]: Why the jump from 3.0.0 to 3.3.0?

Comment (Nayeemuddin-khan) [reply package.json]: I like to update to .3 . Want to revert it back for .1 ?

Comment (xres) [reply package.json]: Unless it's conflicting with published version, versioning should follow semver.
https://semver.org/

**Review by xres — CHANGES_REQUESTED** [review]
Remove .DS_Store from commit.

**Comment by xres** [comment]
@Dianoga could you help review?

File: package.json (hunk 1 above)
Comment (Dianoga) [reply package.json]: I don't see any reason to jump by more than .1

Comment (Nayeemuddin-khan) [reply package.json]: Changed to .1 fo version

**Review by Dianoga — DISMISSED** [review]

**Comment by Dianoga** [comment]
@xres If it looks good to you, I can merge it

**Comment by Dianoga** [comment]
Looks good to me once the .DS_Store file is removed

**Pull request merged**